forms/src/app/login-model-form/login-model-form.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'login-model-form',
+  templateUrl: './login-model-form.component.html',
+  styleUrls: ['./login-model-form.component.css']
+})
+export class LoginModelFormComponent implements OnInit {
+
+  username = "Shivaji";
+  password = "ReadySetGo";
+  remember = true;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+
+  onSubmit(value) {
+    console.log(value);
```

Comment:
As we talk, add a checkbox for phone number, and have the phone fields add and removed dynamically based of the state of the checkbox, thanks!